src/backend/PhotoAPI/BusinessLogic/Services/PhotoService.cs

```diff
@@ -66,6 +74,34 @@ public async Task<IEnumerable<PhotoListDTO>> UploadPhotosAsync(IEnumerable<Photo
 			return createdPhotos;
 		}
 
+		public async Task<IEnumerable<PhotoThumbnailDTO>> CreateThumbnailsToPhotosAsync(IEnumerable<Guid> photosToProcessIds)
+		{
+			IEnumerable<PhotoDocument> photoDocuments = await _elasticService.GetPhotosAsync(photosToProcessIds);
+
+			List<PhotoThumbnailDTO> updatedPhotos = new List<PhotoThumbnailDTO>();
+			foreach (PhotoDocument photoDocument in photoDocuments)
+			{
+				string extension = System.IO.Path.GetExtension(photoDocument.BlobName);
+				byte[] imageBlobFile = await _blobStorage.GetAsync(photoDocument.BlobName);
+
+				photoDocument.Blob64Name = await CreateThumbnailAsync(imageBlobFile, extension, 64);
+				photoDocument.Blob256Name = await CreateThumbnailAsync(imageBlobFile, extension, 256);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=5244170)